ga4gh/serverconfig.py

```diff
@@ -45,6 +58,38 @@ class ProductionConfig(BaseConfig):
     DATA_SOURCE = None
 
 
+class GoogleOIDCConfig(ProductionConfig):
```

Comment:
Perhaps we should have a an OidcConfig superclass, which defines `OIDC_CLIENT_ID = "XXX"` etc. Also, the class should be `GoogleOidcConfig`, as we've made an exception to PEP8 for this (see STYLE.md).